### PR TITLE
Parsing Database info into DataFrame

### PR DESCRIPTION
**Pull request by jbotpoint**
- Parses Database info into dataframe
- Fixed for minor key in user registration

### CODE REVIEW

File: backend/api/views.py [hunk 1]
```diff
@@ -59,17 +61,37 @@ def matchTeams(request):
...
+                    dictOfStudentInfo['interests'].append(interest.interest)
+                
+                dictOfStudentInfo['skills'] = []
+                for skill in profileToAdd.get().skills.all():
+                    dictOfStudentInfo['skills'].append(skill.skill)
+                listOfDictOfStudentInfo.append(dictOfStudentInfo)    
+        #--------------------------    
+        df = pd.DataFrame(listOfDictOfStudentInfo)
+        print(df)
+        #matched_students = match_students(df)
```
Comment (jbotpoint) [inline backend/api/views.py]: This is where we would call the algo

Comment (meryadri) [reply backend/api/views.py]: ok it looks good to me, should @bayazitbora be reviewing this as well?

**Review by meryadri — DISMISSED** [review]
this looks good to me

File: backend/src/algorithm
Comment (meryadri) [inline backend/src/algorithm:1]: Why does this folder have a version of the whole repo, including the frontend and backend again?

**Comment by jbotpoint** [comment]
(copy and pasting again) not sure where you're seeing a whole repo, but this file pulls the algorithm from the commit on github as a package (was an alternative to moving the whole folder when i was trying to get it to work). Though this can be removed. Regardless this branch isn't ready to be merged yet anyway

**Comment by meryadri** [comment]
> (copy and pasting again) not sure where you're seeing a whole repo, but this file pulls the algorithm from the commit on github as a package (was an alternative to moving the whole folder when i was trying to get it to work). Though this can be removed. Regardless this branch isn't ready to be merged yet anyway

the problem is that if you go to VS code and load that branch you will see a copy of the whole repo within that folder, not just the algorithm side, which means that there will be a version of the backend and frontend directories as well...

**Comment by jbotpoint** [comment]
Yeah i dont see the duplicate repo when i pull this branch... but ill push a change to remove this package
image

**Comment by jbotpoint** [comment]
Closing this to avoid confusion

**Pull request closed without merging**